src/dashboard/src/components/access/views.py

```diff
@@ -178,8 +195,22 @@ def record_children(client, request, system='', record_id=''):
     elif request.method == 'GET':
         records = client.get_resource_component_and_children(record_id,
                                                              recurse_max_level=3)
+        records = _get_sip_arrange_children(records, system)
         return helpers.json_response(records['children'])
 
+def _get_sip_arrange_children(record, system):
+    """ Recursively check for SIPArrange associations. """
+    try:
+        mapping = SIPArrangeAccessMapping.objects.get(system=system, identifier=record['id'])
+        record['path'] = mapping.arrange_path
+        record['children'] = record['children'] or []
+        record['has_children'] = True
+    except (SIPArrangeAccessMapping.MultipleObjectsReturned, SIPArrangeAccessMapping.DoesNotExist):
+        pass
+    if record['children']:  # record['children'] may be False
```

Comment:
That would make all ArchivesSpace children directories, not files, which would probably be a good thing, so sure.